### ISSUE TITLE
Error about permission denied when using the docker version. 

### ISSUE THREAD

**Issue by otzfadia** [i2bc/SURFMAP#5]
Hello, I am running the following: 
```
sudo python3 run_surfmap_image.py -d OREN/ -pdb OREN/3fav.pdb -tomap binding_sites
Running: /home/surfmap/MSMS/pdb_to_xyzr /input/3fav.pdb > ./shells/3fav.xyzr-tmp ...

surface property mapping: binding_sites
Traceback (most recent call last):
  File "/home/surfmap/tools/SurfmapTools.py", line 178, in <module>
    main()
  File "/home/surfmap/tools/SurfmapTools.py", line 93, in main
    out = open(outdir + "%s_%s_partlist.out"%(pdbName.split(".")[0],tomap),"w")
PermissionError: [Errno 13] Permission denied: '/output//3fav_bfactor_partlist.out'

Mapping of the residues given in input failed. This can be due to a malformed residue file or a mistake in the reisdue numbering, type or chain.

Format should be the following:
chain residuenumber residuetype
example: A	5 LEU

Exiting now.
```
Why I get permission denied, even when I use sudo? Cheers! Oren

**Comment by otzfadia** [comment]
When I run the manual version, I get different error: 
```
(base) otzfadia@testprox:~/lib/SURFMAP$ sudo python3 surfmap.py -d OREN/ -pdb OREN/3fav_CD.pdb -tomap binding_sites

SURFMAP:    Projection of protein surface features on 2D map
Authors:    Hugo Schweke, Marie-Hélène Mucchielli, Nicolas Chevrollier,
            Simon Gosset, Anne Lopes
Version:    1.5
Copyright (c) 2021, H. Schweke


SURFMAP relies on the use of MSMS (Copyright (c) M. F. Sanner, 1994).
MSMS is free for academic use. For commercial use please contact
M. F. Sanner at sanner@scripps.edu.

If you use SURFMAP for your research, please cite the following
papers:
    - Hugo Schweke, Marie-Hélène Mucchielli, Nicolas Chevrollier,
      Simon Gosset, Anne Lopes (2021) SURFMAP: a software for mapping
      in two dimensions protein surface features. bioRxiv
      2021.10.15.464543; doi: https://doi.org/10.1101/2021.10.15.464543

    - Sanner, M.F., Spehner, J.-C., and Olson, A.J. (1996) Reduced 
      surface: an efficient way to compute molecular surfaces. 
      Biopolymers, Vol. 38., (3), 305-320.

SURFMAP can also optionnaly map electrostatic potential through the use
of the APBS software. So if you use SURFMAP with the option 
'-tomap electrostatics' and find it useful to your research please cite
one or more of the papers listed in the following webpage:
https://apbs.readthedocs.io/en/latest/supporting.html
---------------------------------------------------------------------------

    
Running: /home/itg.be/otzfadia/lib/SURFMAP/MSMS/pdb_to_xyzr OREN/3fav_CD.pdb > ./shells/3fav_CD.xyzr-tmp ...

surface property mapping: binding_sites
Traceback (most recent call last):
  File "/home/itg.be/otzfadia/lib/SURFMAP/tools/SurfmapTools.py", line 5, in <module>
    import Structure
  File "/home/itg.be/otzfadia/lib/SURFMAP/tools/Structure.py", line 4, in <module>
    import os, os.path, sys, string, re, math, numpy
ModuleNotFoundError: No module named 'numpy'

Mapping of the residues given in input failed. This can be due to a malformed residue file or a mistake in the reisdue numbering, type or chain.

Format should be the following:
chain residuenumber residuetype
example: A	5 LEU

Exiting now.
```
Thanks for your help, in advance!

**Comment by otzfadia** [comment]
Thank you very much for your speedy reply! Indeed the pip3 did the trick. I now have a question regarding interpetation of the results. May I contact you directly? Cheers! Oren

**Comment by otzfadia** [comment]
[Archive.zip](https://github.com/i2bc/SURFMAP/files/10347037/Archive.zip)
Files I used.

**Comment by nchenche** [comment]
You're welcome and thank you for the archive, I will take a look. 

You can contact us at the following mails:
- anne.lopes@i2bc.paris-saclay.fr
- hugo.schweke@weizmann.ac.il
- nicolas.chevrollier@laposte.net

Please put all of us in copy to increase the speed of your answer(s).

Cheers

**Comment by nchenche** [comment]
So I looked at your archive. From what I saw, I understand that you tried to predict and map the binding sites of components of a multimeric protein. Unfortunately, SURMAP does not predict binding sites, it only maps them as long as their residues have already been identified and this information been given in the b-factor column of an edited pdb file.

For instance, let's say for simplicity that you know that the residues I, F, and Q of the chain A constitute a binding site. To map those residues with SURFMAP, you will need to edit the pdb file in the following way:
- enter a value of 1 in the b-factor column of each atoms constituting the residues I, F and Q from the chain A
- enter a value of 0 in the b-factor column of each atoms of residues that are not part of the binding site

Hope I didn't misunderstand your aim of using SURFMAP here and answer your question.

I will try (maybe tomorrow) to make a script to help users modify a pdb file from simplified information on the known residues of a binding site in order to avoid editing "manually" a pdb file, if that could help.
